amqp/src/test/java/akka/stream/alpakka/amqp/javadsl/AmqpConnectorsTest.java

```diff
@@ -75,7 +73,7 @@ public void throwIfCanNotConnect() throws Throwable {
         AmqpSink.createSimple(
             AmqpSinkSettings.create(connectionProvider)
                 .withRoutingKey(queueName)
-                .withDeclarations(queueDeclaration));
+                .withDeclarations(Collections.singletonList(queueDeclaration)));
```

Comment:
Would it make sense to have a `withDeclaration` (or `withDeclarations(String)`) to support the singleton case, which seems common?

Reply:
I think that seems to be worth it, yes.